pkg/lib/cockpit-components-listing.jsx

```diff
@@ -41,14 +41,16 @@ require('./listing.css');
  *     to view expanded item details, if not set, navigation isn't available
  * listingDetail optional: text rendered next to action buttons, similar style to the tab headers
  * listingActions optional: buttons that are presented as actions for the expanded item
+ * expanded optional: the entry will be initially rendered as expanded, changes after component creation do not take effect
```

Comment:
These changes to the shared component should be a separate pull request, reviewed separately. Could you split them out?